ci/Jenkinsfile-test

```diff
@@ -0,0 +1,45 @@
+pipeline {
+    agent any
+    options {
+        disableConcurrentBuilds()
+        skipDefaultCheckout()
+    }
+
+    environment {
+        PIP_INDEX_URL='https://artifacts.internal.inmanta.com/inmanta/dev'
+    }
+
+    stages {
+        stage("clear workspace and checkout source code") {
+            steps {
+                deleteDir()
+                checkout scm
+            }
+        }
+        stage("setup virtualenv") {
+            steps {
+                sh'''
+                    python3 -m venv env
+                    env/bin/pip install -U pip poetry pytest
+
+                    # explicitly activate environment so poetry knows to install within it
+                    source env/bin/activate
+
+                    poetry install
+                '''
+            }
+        }
+        stage("run tests") {
+            steps {
+                sh 'sudo env/bin/pytest -v -s --junitxml=junit.xml'
```

Comment:
Also add `junit 'junit.xml'` to export the junit.xml results to jenkins.

Reply:
I get `/home/jenkins/workspace/ols_nfv-test-api_issue_9-jenkins@tmp/durable-5495d7d6/script.sh: line 1: junit: command not found` when I try that. Any idea what could cause this? Am I missing a dependency?

Reply:
Normally I put this in a post-block as mentioned here https://www.jenkins.io/doc/pipeline/tour/tests-and-artifacts/. But it should also work in a build step. Yaron once did that.

Reply:
Oh, it's a jenkins step, that explains. I'll add it.